imgui-opengl3/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+// TODO: Consider making local versions of gl_loader and imgui opengl renderer,
+// but this works for the time being
+pub use {gl_loader, imgui_opengl_renderer};
```

Comment:
This is perfectly ok if it works for current use cases, but yeah, I'd eventually like to embed our own implementations if possible.

Reply:
Agreed :)